docs-api/frontity-packages/features-packages/google-ad-manager.md

```diff
@@ -0,0 +1,168 @@
+| **`unit`**  | string | yes      | The (Google supplied) adUnitPath code for the ad unit to be displayed. _[more info](https://developers.google.com/publisher-tag/reference#googletag.slot-googletag.defineslotadunitpath,-size,-opt_div)_ |
+| **`size`**  | array  | yes      | The width and height to display the ad.                                                                                                                                                                  |
+| `targeting` | array  | no       | One or more keys, each with one or more associated values. _[more info](https://developers.google.com/publisher-tag/guides/key-value-targeting)_.                                                        |
+| `data`      | array  | no       | Other data that you want to pass to the Slot.                                                                                                                                                            |
+
+## Examples
+
+Example with one ad:
+
+```js
+export default {
+  packages: [
+    {
+      name: "@frontity/google-ad-manager",
+      state: {
+        fills: {
+          googleAdManager: {
+            belowHeaderAd: {
+              slot: "Below Header",
+              library: "googleAdManager.GooglePublisherTag",
+              priority: 5,
+              props: {
+                id: "div-gpt-below-header",
+                unit: "/6499/example/banner",
+                size: [320, 100],
+              },
+            },
+          },
+        },
+      },
+    },
+  ],
+};
+```
+
+If you need to add more than one ad you can give each object in the `state.fills.googleAdManager` object it's own key:
+
+```js
+export default {
+  packages: [
+    {
+      name: "@frontity/google-ad-manager",
+      state: {
+        fills: {
+          googleAdManager: {
+            belowHeaderAd: {
+              slot: "Below Header",
+              library: "googleAdManager.GooglePublisherTag",
+              priority: 5,
+              props: {
+                id: "div-gpt-below-header",
+                unit: "/6499/example/banner",
+                size: [320, 100],
+              },
+            },
+            belowContentAd: {
+              slot: "Below Content",
+              library: "googleAdManager.GooglePublisherTag",
+              priority: 5,
+              props: {
+                id: "Below Content",
+                unit: "/6499/example/banner",
+                size: [300, 600],
+                targeting: {
+                  interests: ["sports", "music", "movies"],
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  ],
+};
+```
+
+## How to use
+
+The recommended usage of this component is as in the examples above using the Slot and Fill pattern. The configuration of the fill(s) is done in the `state.fills.googleAdManager` namespace in `frontity.settings.js`.
+
```

Comment:
I would add a quick example (reminder) of how to use the Slot pattern with this AdManager by using the Fill definition explained above

Reply:
Hi @juanmaguitar , I'm not sure what you mean. Do you mean repeat the examples in the "Examples" section in the "How to use" section? Or do you mean move the examples from "Examples" to "How to use"?

Reply:
I've pushed a commit with my suggestions and some extra examples. 
Feel free to modify/remove the content from there